src/specs/VideoNativeComponent.ts

```diff
@@ -286,7 +287,7 @@ export type OnAudioFocusChangedData = Readonly<{
 
 type ControlsStyles = Readonly<{
   hideSeekBar?: boolean;
-  seekIncrementMS?: number;
+  seekIncrementMS?: Float;
```

Comment:
an Int32 shall be enough here ?!

Reply:
Yep, I think it would be better!
if we don't use dots with numbers